boards/st/sensortile_box_pro/doc/index.rst

```diff
@@ -214,6 +217,13 @@ The final application may use it to declare SensorTile.box PRO device as belongi
 certain standard or vendor class, e.g. a CDC, a mass storage or a composite device with both
 functions.
 
+BlueNRG-LP chip
+===============
+
+The board is equipped with a ST Microelectronics `BlueNRG-LP`_ chip. Before using this
+functionality on zephyr a chip f/w upgrade may be necessary. Please follow the information
+on :ref:`sensortile_box_pro_sample_ble_fwupg` sample.
```

Comment:
use `:zephyr:code-sample:` to reference once you've update the sample's readme to use the code-sample directive

Reply:
ack